tools/firmware-utils/src/dlink-sge-image.h

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```

Comment:
Was the original code also MIT? Otherwise you can't do this...

Reply:
not sure what the exact legal relevance of this is, so should I just remove `OR MIT` here?

```
D-Link FOSS (Free Open Source Software)  Statement:

This D-Link product includes software code developed by third parties, including  software code subject to the GNU General Public
License ("GPL") or GNU Lesser General Public License ("LGPL"). As applicable, the terms of the GPL and LGPL, and information on
 obtaining access to the GPL Code and LGPL Code used in this product, are available to you at:

http://tsd.dlink.com.tw/GPL.asp

The GPL/ LGPL Code and any used in this product is distributed WITHOUT ANY WARRANTY and is subject to the copyrights of one
 or more authors. 

For details, please see the terms of the GPL, LGPL and other FOSS License from "SoftwareLicenseDeclaration.pdf".
```

The pdf contains certain Open Source projects and their respective website URLs.

Some GPL releases however just contain the GPL from 1991 in the top directory.

Reply:
Use GPL 2.0 standalone...

Reply:
removed `MIT` and `or-later`